.github/gh-actions-self-hosted-runners/helper-functions/cloud-functions/generateToken/index.js

```diff
@@ -0,0 +1,77 @@
+//  Licensed to the Apache Software Foundation (ASF) under one
+//  or more contributor license agreements.  See the NOTICE file
+//  distributed with this work for additional information
+//  regarding copyright ownership.  The ASF licenses this file
+//  to you under the Apache License, Version 2.0 (the
+//  "License"); you may not use this file except in compliance
+//  with the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing,
+//  software distributed under the License is distributed on an
+//  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//  KIND, either express or implied.  See the License for the
+//  specific language governing permissions and limitations
+//  under the License.
+
+//This function provides registration tokens for windows and linux self-hosted
+//runners, a service account with the appropriated permissions should be used to
+// invoke the Cloud Function.
+
+import functions from '@google-cloud/functions-framework';
+import { Octokit } from "octokit";
+import { createAppAuth } from "@octokit/auth-app";
+
+const REQUIRED_ENV_VARS=["APP_ID","PEM_KEY","CLIENT_ID","CLIENT_SECRET","APP_INSTALLATION_ID","ORG"]
+
+
```

Comment:
```suggestion
const REQUIRED_ENV_VARS=["APP_ID","PEM_KEY","CLIENT_ID","CLIENT_SECRET","APP_INSTALLATION_ID","ORG"]

```

Nit: remove extra space here and elsewhere

Reply:
Removed blank spaces from the `js` scripts, thanks!